starter-code/TicTacToe/app/src/main/java/ly/generalassemb/drewmahrt/tictactoe/GameActivity.java

```diff
@@ -1,13 +1,73 @@
 package ly.generalassemb.drewmahrt.tictactoe;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class GameActivity extends AppCompatActivity {
+    TextView mTurn;
+    TextView mBox, mBox2, mBox3, mBox4,
+            mBox5, mBox6, mBox7, mBox8, mBox9;
+    String mPlayer1, mPlayer2;
+    boolean mNextTurn;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_game);
+
+        mTurn = (TextView) findViewById(R.id.game_message_text);
+        mBox = (TextView) findViewById(R.id.textView);
+        mBox2 = (TextView) findViewById(R.id.textView2);
+        mBox3 = (TextView) findViewById(R.id.textView3);
+        mBox4 = (TextView) findViewById(R.id.textView4);
+        mBox5 = (TextView) findViewById(R.id.textView5);
+        mBox6 = (TextView) findViewById(R.id.textView6);
+        mBox7 = (TextView) findViewById(R.id.textView7);
+        mBox8 = (TextView) findViewById(R.id.textView8);
+        mBox9 = (TextView) findViewById(R.id.textView9);
+
+        // Get the Intent from MainActivity
+        Intent intent = getIntent();
+        mPlayer1 = intent.getStringExtra("Player1");
+        mPlayer2 = intent.getStringExtra("Player2");
+        mTurn.setText(mPlayer1 + "'s Turn");
```

Comment:
You used the Intent and the extras well here! I suggest using constants (`public static final` variables) for the keys rather than hard-coding the keys in two places.